sklearn/linear_model/_bayes.py

```diff
@@ -219,7 +167,7 @@ class BayesianRidge(RegressorMixin, LinearModel):
     """
 
     _parameter_constraints: dict = {
-        "max_iter": [Interval(Integral, 1, None, closed="left"), None],
+        "max_iter": [Interval(Integral, 1, None, closed="left")],
```

Comment:
I directly removed `None` as a valid value. It was added only for the deprecation period and was never meant to be a publicly valid value. It should have been marked as `Hidden` (something to remember for future ones). That's why I figured it wouldn't be necessary to make a new deprecation cycle to remove `None`. What do you think ?

Reply:
Fine with me. I'm might be the culprit.